### ISSUE TITLE
[FEATURE REQUEST] Add some more stats to the autolookup feature

### ISSUE THREAD

**Issue by zephiowl** [BestOfAllCoding/SUPLauncher-Reborn#24]
**Suggestion Type:** Application suggestion

**Describe the solution you'd like**
I would like some more detail to be provided under the PO total; such as money count; karma count; and possible ORG

![image](https://user-images.githubusercontent.com/66909997/183246337-fcf041d6-d5c9-487e-a31c-d3be844cefd4.png)

**Comment by Schmitin** [comment]
Would probably be good to have this extra detail be able to be enabled/disabled in the settings.

**Comment by aidanH39** [comment]
Yes. I did think of this first when I first made this overlay. But it’s more of a staff tool so I didn’t think it would be needed. I may add this in the upcoming future.

**Comment by Schmitin** [comment]
Having the org shown would help staff determine RP relation between players at a glance.
